### ISSUE TITLE
Customise automatic form

### ISSUE THREAD

**Issue by adrianq** [EyeSeeTea/dataset-configuration#73]
- [x] There seems to be a problem with buttons at the bottom of data entry. Haven't seen it myself. Let's keep an eye on that
- [x] Grey fields need to be hidden (just for columns)
- [x] Long category option combination makes table too wide
We can find an example of all this in Dataset: DANIDA OLD / OU: NRC

**Comment by tokland** [comment]
- Long catOptionCombos: There are many ways to do it. What I've done: A category level with too much coc's is broken done to 1 table per catOption of this category. I split the table as many times as needed until the number of COCs in this table is less than N (currently, N=15). 

- Buttons: I've not found any problem, I could set as _complete_, _incomplete_ and validate anytime I tested. I've only fixed the box width to avoid the two rows of buttons (with += 5px)

![dhis2-entry-tables.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/57022c2f494d000e00432e8e/125516c66e82c728ace21e0d46ec9c9330d0ceac81fdad5ae949f196254334316546b3397f8fe62bab0021435a581cfd14595b16aaac8e23f0b23d238a5d09edd7655d70d42103eee9dc4be05ce36ade0d97bcae884acae93825e6838c4d371e011180fa4f78.png)
